packages/svelte/src/internal/client/render.js

```diff
@@ -1353,14 +1365,14 @@ export function bind_this(element_or_component, update, get_value, get_parts) {
 	const e = effect(() => {
 		old_parts = parts;
 		// We only track changes to the parts, not the value itself to avoid unnecessary reruns.
-		parts = get_parts?.() || [];
+		parts = get_parts?.() || (element_or_component ? [element_or_component] : []);
```

Comment:
why do we need the element/component to be part of this? the parts are dynamic bits that point to the value which then contains the element/component, and the element/component shouldn't be necessary here

Reply:
Ah that was some older code. Let me remove.